src/testdir/gen_opt_test.vim

```diff
@@ -1,7 +1,7 @@
 " Script to generate src/testdir/opt_test.vim from src/optiondefs.h and
 " runtime/doc/options.txt
 
-set cpo=&vim
```

Comment:
`:set cpo=&vim` sets 'cpo' to contains only the `&`, `v`, `i`, `m` flags, which isn't correct.